lib/airbrake-ruby/async_sender.rb

```diff
@@ -13,10 +13,9 @@ def initialize(config)
       @unsent = SizedQueue.new(config.queue_size)
       @sender = SyncSender.new(config)
       @closed = false
-      @workers = ThreadGroup.new
+      @workers = nil
 
-      config.workers.times { @workers.add(spawn_worker) }
-      @workers.enclose
+      spawn_workers
```

Comment:
Remove this? It is kinda nice that we don't spawn any workers if async notifier is not used.